extensions/vscode/src/utils/files.ts

```diff
@@ -207,34 +205,6 @@ export function standalonePathSorter(sep: string) {
   };
 }
 
-export function splitFilesOnInclusion(
```

Comment:
This functionality was moved to the util files TypeScript file at the webview level so the files as well as the `includedFiles` and `excludedFiles` list can be in the store.